docs/roadmap.md

```diff
@@ -11,37 +11,49 @@ Here's how we intend to roll out Project Reunion:
 <table>
   <tbody>
     <tr>
-      <th>//BUILD 2020</th>
-      <th>End of 2020</th>
-      <th>2021++</th>
+      <th>Reunion 0.5 (March 2021)</th>
+      <th>Reunion 0.8 (Summer 2021)</th>
+      <th>Reunion 1.0 (Fall 2021)</th>
+      <th>2022 and Beyond</th>
     </tr>
     <tr>
-      <td>
+    <td>
+      <ul>
+            <li>1st supported public release</li>
+            <li>WinUI 3 to work in Desktop Packaged (Win32) Apps</li>
+            <li>Updates to MSIX to easily have their apps updates and repair</li>
```

Comment:
These MSIX updates are coming from the platform, though, not specifically this Reunion repo and its packages.

Also consider instead "have their apps update and self-repair".

Reply:
Yeah if it's not Reunion itself, I'd say remove the MSIX bullet point